backend/quadcore/manager/data.py

```diff
@@ -33,8 +33,23 @@ def set_user_profile(cls, info):
         return cls.db.hmset("user:" + info["email"], info)
 
     @classmethod
-    def is_exist_user(cls, email):
+    def check_email_username(cls, email):
+        """
+        Check whether email and username are connected
+        """
+        return cls.db.hget('email:' + email, 'username')
```

Comment:
Why key's name is `email:`? Let's discuss for more meaningful name: like `user:`.

Reply:
In here, just I wanna distinguish with another key format